core/driver/sources/src/main/resources/reference.conf

```diff
@@ -79,6 +79,16 @@ probe {
     //
     // If you want to use the Ultimate edition, replace "ideaIC" occurences with "ideaIU" in your config file. Community
     // edition ("ideaIC") is the default one.
+    //
+    // You can also add your custom intellij repository on top of the 4 default repositories, so that ide-probe will
```

Comment:
Okay, so given our discussion on Slack, this entire comment is **not** certain at this point, right?

Reply:
Hm, no - my comment is right. But my example is different - in the example from comments we have:
```
intellij.repositories = [
  "file:///"${HOME}"/.my_intellij_idea_downloads/[revision]/ideaIC-[revision][format]",
  official
]
```
so - a pattern that points to a `.zip` or `.dmg` file with intelliJ

Reply:
And here: https://github.com/VirtusLab/git-machete-intellij-plugin/blob/master/src/uiTest/resources/ideprobe.conf#L43 - we have an usage that points to a directory with installed IntelliJ

Reply:
Okay... AFAICS since https://github.com/VirtusLab/ide-probe/pull/179 we're able to pass unzipped folders as `intellij.resolvers.intellij.repositories` right?

Reply:
Let me check that and make sure

Reply:
Ok, you are right - we're able to pass unzipped folders as `intellij.resolvers.intellij.repositories` - example of such successful tests with SNAPSHOT ide-probe version from this PR:
https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1107